organise_desktop/Clean.py

```diff
@@ -3,6 +3,7 @@
 import os
 from cronController import  schedule_end, schedule_start
 from organiseDesktop import undo, organise_desktop
+from os import path, mkdir, listdir, rename, environ, rmdir
```

Comment:
This might be a bit of an overreaction, but Wouldn't it be better to just use os module instead of importing those functions again, just like other contributors did, for the sake of consistency of the code style?

Reply:
Yep, you are right, or just import the needed ones but not both. Give it a go :)